test/helpers/Fixtures.sol

```diff
@@ -156,6 +160,9 @@ contract DeploymentFixtures is StdCheats, Test {
             ".LidoLocator"
         );
         vm.label(deploymentConfig.lidoLocator, "LidoLocator");
+
+        deploymentConfig.gateSeal = vm.parseJsonAddress(config, ".GateSeal");
```

Comment:
`parseJsonAddress` reverts with "no matching value found" for non-existing keys. devnet-0's artifact doesn't have such a key. Do we want to patch it, or it doesn't make sense for us?

Reply:
yea it makes sense